packages/react/src/app-layout/AppLayoutSidebarNav.tsx

```diff
@@ -66,9 +68,10 @@ export function AppLayoutSidebarNav({
 							) : null}
 							{groupItems.map((item, idx) => (
 								<AppLayoutSidebarNavListItem
-									key={idx}
-									item={item}
 									activePath={activePath}
+									background={background}
+									item={item}
+									key={idx}
```

Comment:
Not a request for a change, just a comment - I always read this as "id extended". Lots of value in the extra "n" and "e" 😅

Reply:
Same. I dislike abbreviated variable names

Reply:
There are 164 instances of `idx` so we should clean that up in another PR